ui/src/Bootstrap/index.scss

```diff
@@ -15,4 +15,5 @@ $font-prefix: '~@ibm/plex'; // path to where the font files are
 @include carbon--type-reset();
 
 // Import all other .scss files here
-@import '../Elements/Text/Text.scss';
\ No newline at end of file
+@import '../Elements/Text/Text.scss';
```

Comment:
Have you tried aliases here? I *think* webpack might be smart enough to use the Module aliases we've set up in webpack.config (from moduleAliases.js)

Reply:
I didn't think/know that would work for the scss too! Cool if it did, but there is nothing wrong with what Andy has here otherwise. If it did work, would suggest we change it in a follow up PR

Reply:
Have left as-is for now